packages/cli/src/cmds/beacon/options.ts

```diff
@@ -86,41 +115,58 @@ export const beaconPathsOptions: ICliCommandOptions<IBeaconPaths> = {
     type: "string",
   },
 
-  configFile: {
-    description: "Beacon node configuration file path",
-    type: "string",
-  },
-
   peerStoreDir: {
     hidden: true,
     description: "Peer store directory",
     defaultDescription: defaultBeaconPaths.peerStoreDir,
     type: "string",
   },
 
-  peerIdFile: {
-    hidden: true,
-    description: "Peer ID file path",
-    defaultDescription: defaultBeaconPaths.peerIdFile,
+  logFile: {
+    description: "Path to output all logs to a persistent log file",
     type: "string",
   },
+};
 
-  enrFile: {
-    hidden: true,
-    description: "ENR file path",
-    defaultDescription: defaultBeaconPaths.enrFile,
-    type: "string",
-  },
+interface IENRArgs {
+  "enr.ip"?: string;
+  "enr.tcp"?: number;
+  "enr.ip6"?: string;
+  "enr.udp"?: number;
+  "enr.tcp6"?: number;
+  "enr.udp6"?: number;
```

Comment:
This options are only used by the beacon cmd, there's no point in defining them close to global options